docs/core.md

```diff
@@ -94,16 +95,16 @@ const appJson: AppSchema = {
   pages: [ ... ],
 };
 
-const app = createBatiq(appJson, [loggerMiddleware, multiplayerMiddleware]);
+const app = createBatiq(log(multiplier(undoRedo(appJson))));
```

Comment:
we should't change the api, appJson should be passed as a parameter to createBatiq